src/pages/generate.tsx

```diff
@@ -296,37 +283,51 @@ export default function ClientIdentifierGenerator() {
-                        <FormHelperText>
-                          Recommended. A refresh token can be used to request
-                          new access tokens after they expire. If not requested,
-                          the user will have to re-authenticate after the access
-                          token expires, which will happen quite frequently
-                          (e.g. every 5 minutes or so). Field names: The field
-                          `scope` will have the value `offline_access` set and
-                          the field `grant_types` will have the value
-                          `refresh_token` set.
-                        </FormHelperText>
+                    <Grid container item spacing={1}>
+                      <Grid item>
+                        <FormLabel sx={{ color: "black", fontWeight: 700 }}>
+                          Refresh Tokens
+                        </FormLabel>
+                      </Grid>
+                      <Grid item>
+                        <VerboseSlider
+                          name="useRefreshTokens"
+                          label="Support usage of refresh tokens / offline access"
+                          description="Recommended. A refresh token can be used to request new access tokens
+                            after they expire. If not requested, the user will have to
+                            re-authenticate after the access token expires, which will happen
+                            quite frequently (e.g. every 5 minutes or so). Field names: The field
+                            `scope` will have the value `offline_access` set and the field
+                            `grant_types` will have the value `refresh_token` set."
+                          value={form.values.useRefreshTokens}
+                          onChange={form.handleChange}
+                        />
                       </Grid>
                     </Grid>
                     <Grid container item>
-                      <Accordion>
-                        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-                          Additional information displayed to users
+                      <Accordion sx={{ boxShadow: "none" }}>
+                        <AccordionSummary
+                          sx={{
+                            paddingLeft: 0,
+                            "&.Mui-expanded .ExpandIcon": {
+                              transform: "rotate(-180deg)",
+                            },
+                            ".ExpandIcon": {
+                              transition:
+                                "transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms",
+                            },
+                          }}
+                        >
+                          <Grid container>
+                            <Grid item>
+                              <Typography variant="h3">More fields</Typography>
```

Comment:
Out of curiosity, why change from the more specific "Additional information displayed to users" and "Advanced OIDC options" to the more generic "More fields" ?

Reply:
Good question! That was on Kyra's suggestions. Me not being a designer, I only adopted it :D

Reply:
I didn't see the updated field names from @NSeydoux, but I've put them back into the designs because they are better!
![image](https://user-images.githubusercontent.com/11729241/191992853-0c10946e-9a4c-48da-b84a-588048cccda9.png)
Expanded: https://sketch.com/s/d5a3607f-eb32-4914-91a2-8d4469ed4cbb
Collapsed: https://sketch.com/s/d5a3607f-eb32-4914-91a2-8d4469ed4cbb